app/routes/planets_route.py

```diff
@@ -0,0 +1,96 @@
+from flask import Blueprint, abort, make_response, request, Response
+from ..db import db
+from app.models.planets import Planet
+
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
```

Comment:
Remember that we use our blueprints for every route that we write. While we don't have too many routes here, they can add up in more robust databases. It might not seem like much, but writing `planets_bp` over and over again can be quite cumbersome! 

One way we get around this is with the naming convention of just `bp`. As long as we separate out our routes into different files, we can name all our blueprints `bp`. To keep our project from getting confused, we can go ahead an alias the different bps when we import them into our __init__.py. This would look like:

     from .routes.planets_route import bp as planets_bp